.github/ISSUE_TEMPLATE/help_desk.md

```diff
@@ -0,0 +1,41 @@
+---
+name: Help Desk
+about: Despite multiple attempts you repeatedly fail to solve an issue?  
+title: "[Help Desk]"
+labels: help desk
+assignees: ''
+
+---
+ 
+## Please note
+<!-- A brief description of what you expected to happen. -->
+- We may ignore any issues due to a lack of time and apologize.
```

Comment:
Please remove this line. We could rephrase it but removal might be best for now.